cabby/data/metagraph/utils.py

```diff
@@ -22,7 +22,7 @@
 from cabby.geo.map_processing.map_structure import Map
 import cabby.geo.util as util
 from cabby.data.wikidata import query
-from cabby.geo.regions import Region
+from cabby.geo.regions import Region, get_region
```

Comment:
Let's not import functions as per the style guide:

https://google.github.io/styleguide/pyguide.html#22-imports

We've been importing some classes like Region and shapely's Point and Polygon when they are used a lot for type annotations. Technically we should do this:

```
from cabby.geo import regions

Region = regions.Region
```

Perhaps we should just go ahead switch over to doing this and follow the import guidelines to stay consistent with code review expectations.

Reply:
Fixed by af6ff62

Reply:
Fixed by af6ff629a331a9ba6fbbf95b311129180f8df03e